tool/tctl/common/inventory_command.go

```diff
@@ -216,11 +231,12 @@ func (c *InventoryCommand) List(ctx context.Context, client *authclient.Client)
 		NewerThanVersion: vc.Normalize(c.newerThan),
 		ExternalUpgrader: upgrader,
 		NoExtUpgrader:    noUpgrader,
+		UpdateGroup:      c.updateGroup,
```

Comment:
I don't think there's a good way around this without defining a new RPC (with better forwards compatibility) but keep in mind that the update group filtering will just not happen if a newer tctl ends up talking to an auth that doesn't know about this field, so maybe we should also filter the list on the client side?